tests/Feature/AccountTest.php

```
@@ -44,6 +46,27 @@ protected function setUp(): void
+    protected function resourceAuthMemberRouteParams($resource): array
+    {
+        return ['type' => 'account', 'account_entity' => $resource->id];
+    }
+
+    protected function resourceAuthSupportsDestroy(): bool
+    {
+        // account-entity.destroy does not exist.
+        return false;
+    }
+
+    protected function createResourceForAuthTest(User $user): AccountEntity
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)